Refactor(eos_cli_config_gen)!: Rearrange eos_cli as per EOS 4.35.0F part 10 (#6310)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

Reviewer: ```suggestion
### `dynamic_prefix_lists[].match_map` is not required anymore
```

      ipv6: IPV6_PREFIX_LIST
```

> ### `roles[].sequence_numbers[].sequence`, `roles[].sequence_numbers[].action` and `roles[].sequence_numbers[].command` now required to generate the configuration and documentation for `roles`

Reviewer: ```suggestion
### Changes to the `roles[].sequence_numbers` data model
```

As of AVD 6.0.0, `roles[].sequence_numbers[].sequence`, `roles[].sequence_numbers[].action` and `roles[].sequence_numbers[].command` must be set to generate the configuration and documentation for `role` configurations.

```diff
 roles:
   - name: NMYROLE
     sequence_numbers:
-      - action: permit
+      - sequence: 10
+        action: permit

>          command: "show plop"

Reviewer: ```suggestion
         command: "show running-config"
```

-      - action: deny
+      - sequence: 20
+        action: deny

>          command: "show plop"

Reviewer: ```suggestion
         command: "show interfaces description"
```
